tests/test_flows/test_simple_flow.py

```diff
@@ -48,10 +48,24 @@ def test_full_load_time_min():
     price_pattern = [i for i in range(11)]
 
     flow = solph.flows.Flow(
-        nominal_value=2,
+        nominal_capacity=2,
         variable_costs=price_pattern,
         full_load_time_min=4.5,
     )
     flow_result = list(_run_flow_model(flow)["flow"][:-1])
 
     assert flow_result == pytest.approx(4 * [2] + [1] + 5 * [0])
+
+
+# --- BEGIN: The following code can be removed for versions >= v0.7 ---
+def test_nominal_capacity_warning():
+    with pytest.warns(FutureWarning, match="nominal_capacity"):
```

Comment:
I guess it would be more concise to match for `nominal_value` as for the storage above, where the test checked for the previous attribute `nominal_storage_capacity`.